src/pages/nav.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
```

Comment:
I left you a review on your navbar implementing you can find the one you other branched